library/control/src/modules/ProductProfile.rb

```diff
@@ -99,7 +100,7 @@ def IsCompliant(productId)
           tmp_path = Ops.add(Ops.add(@profiles_dir, name), ".profile")
           SCR.Execute(
             path(".target.bash"),
-            Builtins.sformat("/bin/cp -a '%1' '%2'", profile, tmp_path)
+            Builtins.sformat("/bin/cp -a '%1' '%2'", profile.shellescape, tmp_path.shellescape)
```

Comment:
Bad idea: This will fail. %1 and %2 are already quoted in that command!
Simply leave it as it is in this particular code location.

Reply:
you are right. I will just remove that quoting. Thanks for catching